# **P4DS Final Project: An Analysis On Global Warming**
Huy G. Tong $^{1 *}$ \
$^{1}$ *Faculty of Information Technology, VNUHCM - University of Science, Vietnam*\
$^{*}$ *Student ID: 21127307*


#### **ABSTRACT**
To be written later.

#### **1. INTRODUCTION**

To be written later.

#### **2. METHOD**

In this section, I will describe in detail the steps used to perform the analysis on the dataset. I will also give a brief description of the data used in this project, as well as include the codes to help derive insightful information.

First, let us import the necessary packages, be sure to run the bellow cell at least once when executing this Notebook.


In [7]:
import pandas as pd 
import numpy as np 
import matplotlib.pyplot as plt
import datetime as dt

#### **2.1 Data**

In [56]:
globtemp = pd.read_csv("data/GlobalTemperatures.csv")
globtemp.sample(5)

dt  LandAverageTemperature  LandAverageTemperatureUncertainty  \
499   1791-08-01                  13.744                              0.807   
1452  1871-01-01                   1.447                              0.793   
1320  1860-01-01                   2.077                              0.860   
2690  1974-03-01                   5.190                              0.114   
1374  1864-07-01                  14.162                              0.570   

      LandMaxTemperature  LandMaxTemperatureUncertainty  LandMinTemperature  \
499                  NaN                            NaN                 NaN   
1452               7.962                          1.379              -2.937   
1320               9.772                          2.260              -2.059   
2690              10.966                          0.134              -0.495   
1374              18.831                          1.333               7.413   

      LandMinTemperatureUncertainty  LandAndOceanAverageTemperature  \
499                             NaN                             NaN   
1452                          1.432                          13.089   
1320                          1.590                          13.170   
2690                          0.098                          14.267   
1374                          0.846                          16.715   

      LandAndOceanAverageTemperatureUncertainty  
499                                         NaN  
1452                                      0.287  
1320                                      0.294  
2690                                      0.054  
1374                                      0.256

#### **2.2 Preprocessing**

In [57]:
globtemp["dt"] = pd.to_datetime(globtemp["dt"], format="%Y-%m-%d")

#### **2.3 Analysis**

note questions:
- how has the global temperature changed throughout the years?
- temp and gas: what is the relation?

#### **3. CONCLUSION**

#### **REFERENCES**